controller/src/main.cpp

```diff
@@ -113,19 +139,19 @@ static void controller_loop() {
     DEV_MODE_comms_handler();
 #endif
 
+    // get sensor readings
```

Comment:
Delete comment; it's obvious from the code.

Reply:
sure... this is a remnant of having the whole get_sensor_readings routine here instead of a dedicated function in sensors.cpp